jerry-core/ecma/builtin-objects/ecma-builtin-array.inc.h

```diff
@@ -19,6 +19,8 @@
 
 #include "ecma-builtin-helpers-macro-defines.inc.h"
 
+#ifndef CONFIG_DISABLE_ARRAY_BUILTIN
+
```

Comment:
To tell the truth I am not fond of modifying a lot of files just to make a script work. I always work in the opposite way. But since these are just ifdefs, probably would not hurt.